src/components/entity/state-badge.ts

```diff
@@ -220,7 +220,7 @@ export class StateBadge extends LitElement {
           position: relative;
           display: inline-flex;
           width: 40px;
-          color: var(--state-icon-color);
+          color: var(--state-icon-color, #44739e);
```

Comment:
We don't need to set this because it's set in `ha-styles`
```suggestion
          color: var(--state-icon-color);
```